docs/_snippets/checkbox-story.md

```diff
@@ -9,27 +7,41 @@ import * as CheckboxStories from './Checkbox.stories';
 
 # Checkbox
 
-A checkbox is a square box that can be activated or deactivated when ticked. 
+A checkbox is a square box that can be activated or deactivated when ticked.
 
 Use checkboxes to select one or more options from a list of choices.
 
 <Canvas of={CheckboxStories.Unchecked} />
-````
+```
 
-```md renderer="react" language="mdx"
-{/* Checkbox.mdx */}
+```mdx filename="Checkbox.mdx" renderer="svelte" language="mdx" tabTitle="Svelte CSF"
```

Comment:
I'm not sure this is picked up in the docs, I still see the "common" code snippet above rendered, even when Svelte is selected.